### PR TITLE
Adds response to functions-ins-and outs

### PR DESCRIPTION
**Pull request by jdaly04**
Attempts the five questions from functions-ins-and
outs diagnostic.
Comfort: 2
Clarity: 2
I used google, notes, and referenced the solution
branch on the Github repo.

### CODE REVIEW

File: lib/diagnostic.js [hunk 1]
```diff
@@ -8,22 +8,32 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+  let result = Math.min();
```
Comment (neugierige) [inline lib/diagnostic.js:11]: Did you look up the definition/syntax for `Math.min()`? Never copy code blindly. Just Google "Javascript math.min()". 

Please study this solution:

```
const min = function min() {
  let result = arguments[0];
  for (let i = 1; i < arguments.length; i++) {
    if (arguments[i] < result) {
      result = arguments[i];
    }
  }

  return result;
};
```

**Comment by neugierige** [comment]
1/5. Hi Julia, please make sure you really understand your solution for `sum`. Because you could have used the same strategy for `min`. As for the other questions, please look over the solution branch, and review the last few lessons on JS. Let me know if you need a 1:1.

**Comment by jdaly04** [comment]
Hi Lu,

Could we have a 1-1 today? Not sure how the schedule will work given the
overtime on our last lesson.

Let me know!
Julia

**Comment by neugierige** [comment]
Unfortunately, I'm already booked for today. Jason is out sick, and Ben will be leaving early too. Check with Rachel and Antony. If neither are available, we can try for tomorrow at 4pm.

**Pull request closed without merging**